packages/electron/lib/electron.js

```diff
@@ -126,7 +126,11 @@ module.exports = {
         const opts = minimist(argv)
 
         if (opts.inspectBrk) {
-          argv.unshift('--inspect-brk=5566')
+          if (process.env.CYPRESS_DOCKER_DEV_INSPECT_OVERRIDE) {
+            argv.unshift(`--inspect-brk=${process.env.CYPRESS_DOCKER_DEV_INSPECT_OVERRIDE}`)
+          } else {
+            argv.unshift('--inspect-brk=5566')
+          }
```

Comment:
I looked at passing this in through the cli as `inspect-brk=0.0.0.0:5566` or `inspect-brk 0.0.0.0:5566` but decided that was more trouble than it was worth since we only need to change it in the even that we want to debug from within a docker container.